apps/docs/src/.vitepress/components/ComponentCard.vue

```diff
@@ -0,0 +1,61 @@
+<script lang="ts" setup>
+export type ComponentCardProps = {
+  /** Component name. */
+  name: string;
+  /** If true an "Implemented" badge will be shown, "Coming Soon" otherwise. */
+  implemented?: boolean;
+};
+
+const props = defineProps<ComponentCardProps>();
+</script>
+
+<template>
+  <div class="card">
+    <article class="card__content">
+      <Badge
+        v-if="props.implemented"
+        class="card__badge card__badge--implemented"
+        text="Implemented"
+      />
+      <Badge v-else class="card__badge" text="Coming Soon" type="info" />
+      <h4 class="card__title">{{ props.name }}</h4>
+    </article>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.card {
+  display: block;
+  border: 1px solid var(--vp-c-default-soft);
+  border-radius: 12px;
+  height: 100%;
+  background-color: var(--vp-c-bg);
+  transition:
+    border-color 0.25s,
+    background-color 0.25s;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    padding: 1.5rem;
+    height: 100%;
+  }
+
+  &__title {
+    font-size: 1.5rem;
+    line-height: 1.5rem;
```

Comment:
seems to have no effect
```suggestion
```

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/9e1cad8c-c0a4-431d-bdcf-46aee66278a8)
ah, because it's already covered by global styles